### ISSUE TITLE
Crash on Undercover (Update 108)

### ISSUE THREAD

**Issue by Afbg123** [JamesWilko/GoonMod#137]
Seems to be a crash caused by Goonmod related to enemy spawning in the latest update.

Application has crashed: C++ exception
[string "lib/managers/group_ai_states/groupaistatebe..."]:404: attempt to index field '_tweak_data' (a nil value)

SCRIPT STACK

---

Callstack:

```
     payday2_win32_release  (???)     ???                                                 
     payday2_win32_release  (???)     ???                                                 
     payday2_win32_release  (???)     ???                                                 
     payday2_win32_release  (???)     zip_get_name                                        
                     ntdll  (???)     RtlFreeHeap                                         
                     ntdll  (???)     ZwQueryVirtualMemory                                
                KERNELBASE  (???)     VirtualQueryEx                                      
                KERNELBASE  (???)     SetUnhandledExceptionFilter                         
                KERNELBASE  (???)     SetUnhandledExceptionFilter                         
     payday2_win32_release  (???)     ???                                                 
     payday2_win32_release  (???)     zip_get_name                                        
```

**Comment by Kilobytz** [comment]
I also get this same error on Train Heist, and I think all heists TBH.

**Comment by ryoleo** [comment]
As far as I can tell it seem to be related to snipers, and maybe swat van turrets. First tested on first world bank, and it seem to crash right as they spawn or take aim. Did a second test  on four stores, to see if was just the snipers, and it crash right as things got loud. the crash log end up a bit different

[string "lib/managers/group_ai_states/groupaistatebe..."]:3311: attempt to index field '_tweak_data' (a nil value) 

basically the same line but a 3311 instead of 404. btw I tested the snipers with the mod disable and they spawn perfectly fine but hadn't tested the swat van with out mods yet.(wasted to much time on this already) also hadn't tested silent runs because I'm not that good at that. Hopefully this help narrow down the error search.

**Comment by Afbg123** [comment]
Until Goonmods updated, here's a tempoary fix:

Go to PAYDAY 2 > mods > Goonmod > goonbase.lua. Go to line 42 ["lib/managers/group_ai_states/groupaistatebesiege"] = "GroupAIStateBesiege.lua", and put 2 dashes in front of it so it becomes --["lib/managers/group_ai_states/groupaistatebesiege"] = "GroupAIStateBesiege.lua",

Will disable mods that affect guard spawns and mutators however.

**Comment by poosyfarts** [comment]
Thanks @Afbg123, that did fix the instant crash.

However, I just crashed on Murky Station. Haven't crashed for a long time. However, here is the crash log proving it was a GoonMod fault:

> Current thread: Main
Script stack:
                      update()  lib/units/beings/player/states/playerstandard.lua:332  
                      update()  @mods/GoonMod/lua/PlayerStandard.lua:42             
                                lib/units/beings/player/playermovement.lua:264      
                            ()  =(tail                                              
                      call):-1  ()                                                  
                        =(tail                                                      
                      call):-1  
-------------------------------

**Comment by iguaan** [comment]
Thanks, @Afbg123. That fixed the instant crashes, however I just crashed at random on Murky Station. The crashlog blames it on GoonMod, see below.

`-------------------------------

Callstack:

         payday2_win32_release  (???)     ???                                                 


-------------------------------

Current thread: Main
Script stack:
                      update()  lib/units/beings/player/states/playerstandard.lua:332  
                      update()  @mods/GoonMod/lua/PlayerStandard.lua:42             
                                lib/units/beings/player/playermovement.lua:264      
                            ()  =(tail                                              
                      call):-1  ()                                                  
                        =(tail                                                      
                      call):-1  
-------------------------------`

Or for an easier read, here's a picture: http://i.imgur.com/zQWMTJt.png

**Comment by JamesWilko** [comment]
Should be fixed in r32. @iguaan please create a separate issue if you are still experiencing your crash.
